src/components/ui/GrommetUI.jsx

```diff
@@ -0,0 +1,544 @@
+                    }}
+                />
+                {valueMultiple.map((item, i) => (
+                    <Card margin='medium' background='light-1' key={i}>
+                        <Box direction='row' align='center'> 
+                            <Avatar margin='small' background='accent-4' size='medium' round={false}>
+                                {item[0]}{item[8]}
+                            </Avatar>
+                            <Box>
+                                <Heading margin='0' level='4'>
+                                    {item}
+                                </Heading>
+                                <Heading margin='0' color='status-unknown' level='6'>
+                                    Not reminded
+                                </Heading>
+                            </Box>
+                        </Box>
+                    </Card> 
+                ))}
+            </Box>
+        </Box>
+)};
+
+const EditAction = props => {
+    const defaultOptions = [];
+    for (let x = 1; x <= 10; x += 1) {
+        defaultOptions.push(`Campaign ${x}`);
+    }
+    const [options, setOptions] = useState(defaultOptions);
+    const [value, setValue] = React.useState('');
+    const [valueMultiple, setValueMultiple] = useState([]);
+    return (
+        <Box fill='horizontal' margin='medium'>
+            <Heading level='3'>
+                Edit action
+            </Heading>
+            <Heading level='6'>
+                Campaign
+            </Heading>
+            {/* It's possible to limit selection */}
+            <Select
+                multiple
+                size='medium'
+                placeholder='Start typing to find or create'
+                value={valueMultiple}
+                options={options}
+                onChange={({ value: nextValue }) => setValueMultiple(nextValue)}
+                onClose={() => setOptions(defaultOptions)}
+                onSearch={text => {
+                    // The line below escapes regular expression special characters:
+                    // [ \ ^ $ . | ? * + ( )
+                    const escapedText = text.replace(/[-\\^$*+?.()|[\]{}]/g, '\\$&');
+                    // Create the regular expression with modified value which
+                    // handles escaping special characters. Without escaping special
+                    // characters, errors will appear in the console
+                    const exp = new RegExp(escapedText, 'i');
+                    setOptions(defaultOptions.filter(o => exp.test(o)));
+                }}
+            />
+            <Heading level='6'>
+                Date
+            </Heading>
+            <DateInput
+                margin={{'horizontal': 'medium'}}
+                format='mm/dd/yyyy'
+                value={(new Date()).toISOString()}
+                onChange={({ value }) => {}}
+            />
+            <Box direction='row'>
+                <Box>
```

Comment:
Is there a reason for not implementing a component TimePicker so that you don't have to repeat the same code twice?